docs/TPMmodulePinout.md

```diff
@@ -0,0 +1,100 @@
+[User Manual B450-F](https://www.bhphotovideo.com/lit_files/403638.pdf)
+![Flash](images/pinouts/Asus-MAXIMUS_IX_FORMULA.svg)
+
+## 5. Asus TPM-M R2.0, Z97 PRO GAMER
+Interface: LPC</br>
+[User Manual TPM-M](https://www.manualslib.com/manual/1447187/Asus-Tpm-M-R2-0.html)</br>
+[User Manual Z97 PRO GAMER](https://dlcdnets.asus.com/pub/ASUS/mb/LGA1150/Z97-PRO_GAMER/E10265_Z97-PRO_GAMER_Guide_v2_web_hi_res.pdf)
+![Flash](images/pinouts/Asus-TPM-M_R2.0.svg)
+
+## 6. Asus-ROG STRIX X570-F GAMING
+Interface: SPI</br>
+[User Manual](https://gamingprofy.com/wp-content/uploads/2021/06/E15827_ROG_STRIX_X570-F_GAMING_UM_v2_WEB.pdf)
+![Flash](images/pinouts/Asus-ROG_STRIX_X570-F_GAMING.svg)
+
+## 7. Asus-B550 PLUS TUF GAMING
+Interface: SPI</br>
+[User Manual](https://dlcdnets.asus.com/pub/ASUS/mb/SocketAM4/TUF_GAMING_B550-PLUS/E16576_TUF_GAMING_B550-PLUS_UM_WEB.pdf)
+![Flash](images/pinouts/Asus-B550-PLUS.svg)
+
+## 8. Gigabyte GA Z77X D3H, GA Z77 D3H, GA 970A-UD3P
+Interface: LPC</br>
+[User Manual Z77X D3H](https://download1.gigabyte.com/Files/Manual/mb_manual_ga-z77x-d3h_e.pdf)</br>
+[User Manual Z77 D3H](https://download1.gigabyte.com/Files/Manual/mb_manual_ga-z77x-d3h_e.pdf)</br>
+[User Manual 970A-UD3P](https://download1.gigabyte.com/Files/Manual/mb_manual_ga-970a-ud3p_e.pdf)
+![Flash](images/pinouts/Gigabyte-GA-Z77X-D3H.svg)
+
+## 9. Gigabyte GA A320M-S2H AMD A320, GA B250 HD3P, B450 Aorus Pro
+Interface: LPC</br>
+[User Manual A320M-S2H](https://gzhls.at/blob/ldb/0/a/1/c/f15c2f507a34fa4fb5f2d78875ef5477838c.pdf)</br>
+[User Manual B250 HD3P](https://download.gigabyte.com/FileList/Manual/mb_manual_ga-b250-hd3p_e.pdf)</br>
+[User Manual B450 AORUS PRO](https://download.gigabyte.com/FileList/Manual/mb_manual_b450-aorus-pro-wifi_1002_e_190528.pdf)
+![Flash](images/pinouts/Gigabyte-GA-A320M-S2H-AMD-A320.svg)
+
+## 10. MSI B75MA E33, MSI B85M E45, MSI B350 TOMAHAWK, MSI X99A GAMING 9 ACK
+Interface: LPC</br>
+[User Manual B75MA E33](https://www.msi.com/Motherboard/b75ma-e33/support#down-manual)</br>
+[User Manual B85M E45](https://pl.msi.com/Motherboard/B85M-E45/support#down-manual)</br>
+[User Manual TOMAHAWK](https://pl.msi.com/Motherboard/B350-TOMAHAWK/support#down-manual)</br>
+[User Manual X99A](https://www.msi.com/Motherboard/x99a-gaming-9-ack/support#down-manual)
+![Flash](images/pinouts/MSI-B75MA-E33.svg)
+
+## 11. MSI Z590 PRO WiFi
+Interface: SPI</br>
+[User Manual Z590](https://www.msi.com/Motherboard/Z590-PRO-WIFI/support#down-manual)
+![Flash](images/pinouts/MSI-Z590_PRO_WiFi.svg)
+
+## 12. Gigabyte Z590 AORUS MASTER
+Interface: SPI</br>
+[User Manual](https://download.gigabyte.com/FileList/Manual/mb_manual_z590-aorus-master_1002_e.pdf)
+![Flash](images/pinouts/Gigabyte-Z590_AORUS_MASTER.svg)
+
+## 13. Supermicro MBD M12SWA
+Interface: LPC</br>
+[User Manual](https://www.supermicro.com/Aplus/support/resources/results.cfm)
+![Flash](images/pinouts/Supermicro_MBD_M12SWA.svg)
+
+## 14. Supermicro X10DAL-i
```

Comment:
The difference is SMB. In our case it does not care, since we will not connect anything there.  believe from LPC perspective this pinout can be compatible with `13. Supermicro MBD M12SWA`.

Reply:
13 and 14 are compatible between themselves and compatbile with 3 8 and 9 if we look closer.

Reply:
ok